appengine/test/tc_sample_apps.rb

```diff
@@ -32,7 +32,7 @@ def run_app_test(dirname)
     ::Dir.chdir(::File.join(APPS_DIR, dirname)) do |dir|
       build_docker_image("", dirname) do |image|
         run_docker_daemon("-p 8080:8080 #{image}") do |container|
-          assert_cmd_output("curl -s -S http://127.0.0.1:8080/", "ruby app", 10)
+          assert_cmd_output("docker exec #{container} curl -s -S http://127.0.0.1:8080", "ruby app", 10)
```

Comment:
do the tests actually fail for you? when I run them, it usually takes a few tries before the container is fully up and serving requests, but eventually the client gets a response.

```
Successfully built 801de2b1f753
docker run --name ruby-test-container-generic-c4331d1f -d -p 8080:8080 ruby-test-image-rails4_app-e7aece60
367703c1f169f81e84c00e17775b573df60a8dcd5bee5632e50a714835a70e18
docker exec ruby-test-container-generic-c4331d1f curl -s -S http://127.0.0.1:8080
curl: (7) Failed to connect to 127.0.0.1 port 8080: Connection refused
...expected result did not arrive yet (iteration 0)...
docker exec ruby-test-container-generic-c4331d1f curl -s -S http://127.0.0.1:8080
curl: (7) Failed to connect to 127.0.0.1 port 8080: Connection refused
...expected result did not arrive yet (iteration 1)...
docker exec ruby-test-container-generic-c4331d1f curl -s -S http://127.0.0.1:8080
docker kill ruby-test-container-generic-c4331d1f
ruby-test-container-generic-c4331d1f
docker rm ruby-test-container-generic-c4331d1f
ruby-test-container-generic-c4331d1f
docker rmi ruby-test-image-rails4_app-e7aece60
Untagged: ruby-test-image-rails4_app-e7aece60:latest
```